boskos/mason/mason.go

```diff
@@ -119,7 +120,12 @@ func CheckUserData(res common.Resource) (common.LeasedResources, error) {
 	}
 
 	if err := res.UserData.Extract(LeasedResources, &leasedResources); err != nil {
-		return nil, err
+		var legacyLeasedResources common.LegacyLeasedResource
+		if err := res.UserData.Extract(LeasedResources, &legacyLeasedResources); err != nil {
+			return nil, err
+		}
+		leasedResources = common.LeasedResources{}
+		leasedResources[""] = legacyLeasedResources
```

Comment:
should we document this behavior somewhere?

Reply:
does prow/ANNOUNCEMENTS.md sound like a good place?

Reply:
well, it's mostly relevant to anyone implementing the Mason interface.

that said, noting the new `Destruct` method on the interface might be worth putting in the announcements.

Reply:
done